web/server/models.py

```diff
@@ -43,6 +43,16 @@ class EventName(str, enum.Enum):
     PLAN_CANCEL = "plan-cancel"
 
 
+class Modules(str, enum.Enum):
+    EDITOR = "editor"
+    DOCS = "docs"
+    PLANS = "plans"
+    PLAN_ACTIVE = "plan-active"
```

Comment:
What is the plan active module?

Reply:
so intention was to display running  plan or plan apply as separate `module` so one can have `plans` module disabled but still be able to see running tasks and updates.  But it actually just placeholder , i remove it for now and add this option as separate PR